outputs.tf

```diff
@@ -12,3 +12,23 @@ output "sa_roles" {
   description = "All roles (except sensitive roles filtered by the module) that are attached to the ServiceAccount of this VM."
   value       = module.service_account.roles
 }
+
+output "self_link" {
+  description = "The URI of the created resource."
+  value       = google_compute_instance.vm_instance.self_link
+}
+
+output "id" {
+  description = "an identifier for the resource with format projects/{{project}}/zones/{{zone}}/instances/{{name}}"
+  value       = google_compute_instance.vm_instance.id
+}
+
+output "instance_id" {
+  description = "The server-assigned unique identifier of this instance."
+  value       = google_compute_instance.vm_instance.instance_id
+}
+
+output "instance_name" {
+  description = "The generated name of the GCloud VM Instance"
```

Comment:
Can you add example format here also like you added an example format for `id` ?

Reply:
Done